.github/renovate.json

```diff
@@ -104,6 +103,10 @@
       "matchPackageNames": ["oss/binaries/kubernetes/kubernetes-node", "oss/binaries/kubernetes/azure-acr-credential-provider", "oss/binaries/deislabs/containerd-wasm-shims"],
       "extractVersion": "^(?P<version>.*?)-[^-]*-[^-]*$"
     },
+    {
+      "matchPackageNames": ["moby-runc", "moby-containerd"],
```

Comment:
seems weird that renovate makes you have duplicate entries for `extractVersions` and `matchUpdateTypes`...

Reply:
ya. Don't know what the logic behind. Otherwise it will report error `packageRules cannot combine both matchUpdateTypes and extractVersion`